odk1-src/aggregate-forms.rst

```diff
@@ -53,6 +53,11 @@ From the :ref:`form list <aggregate-view-blank-form-list>`,  click on form's :gu
    :alt: Image showing xml viewer for form.
 
 .. _aggregate-toggle-form-download:
+
```

Comment:
Maybe wrap this in a note?

Reply:
Done! I've also fixed the label on line 55, which should be below the text I've added :P (RST is hard!)